tests/testing_data/metadata.json

```diff
@@ -1,5 +1,5 @@
 {
-    "schema": "https://github.com/Project-MONAI/MONAI-extra-test-data/releases/download/0.8.1/meta_schema_202203171008.json",
+    "schema": "https://github.com/Project-MONAI/MONAI-extra-test-data/releases/download/0.8.1/meta_schema_20220324.json",
```

Comment:
I checked the schema file, seems you forget to update the key to verify:
```
"authorship": {
    "description": "state author(s) of the model package.",
    "type": "string"
}
```

Thanks.